packages/health-check/Dockerfile

```
@@ -30,9 +30,10 @@ ENV NODE_ENV production
 # 3. start crond in the background to schedule periodic health checks
 # 4. start the health-check api service
 CMD [ "sh", "-c", \
-      "serverip=$(node src/whatismyip.js) ; \
-       dnsmasq --no-resolv --log-facility=/var/log/dnsmasq.log --address=/$PORTAL_DOMAIN/$serverip --server=127.0.0.11 ; \
-       echo \"$(sed 's/127.0.0.11/127.0.0.1/' /etc/resolv.conf)\" > /etc/resolv.conf ; \
-       crond ; \
+      "serverip=$(node src/whatismyip.js) && \
+       echo export serverip=${serverip} >> /etc/environment && \
```

`nit` single quotes couldn't hurt here

---

it's double quotes that we want here, otherwise `${serverip}` would not get substituted for the actual ip

```suggestion
       echo "export serverip=${serverip}" >> /etc/environment && \
```